src/components/accordion/index.md.njk

```diff
@@ -17,83 +17,124 @@ The accordion component lets users show and hide sections of related content on
 
-The design for this component was originally created and tested by a team at the Government Digital Service (GDS) on the [GOV.UK Service Manual](https://www.gov.uk/service-manual) and [GOV.UK topic pages](https://designnotes.blog.gov.uk/2017/06/29/designing-new-navigation-elements-for-gov-uk/).
+[Read about the research and development that went into improving the accordion component](https://github.com/alphagov/govuk-design-system-backlog/issues/1).
 
 The team made sure the component is accessible, for example that users can interact with it using just the keyboard.
 
 ### Known issues and gaps
 
-The section headings can be mistaken for links, but are treated as buttons. This fails [WCAG 2.1 success criterion 1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships). We plan to fix this issue by the end of June 2021.
 
-The plus and minus icon is on the right side of the component, which means users of screen magnifiers might not see it.
+#### Summary lines
+
+Adding a summary line with more than a few short words will likely make the button text too long, particularly for users of screen readers.```
+
+We need to better understand when service teams use summary lines and how this affects users of screen readers. [Share your findings and research with us](https://github.com/alphagov/govuk-design-system-backlog/issues/1).
+
+However, summary text in accordions is widely used by services so we want to get feedback to inform us what to do next.
 
-The 'Open all' button reads out as 'Open all sections' for screen readers. This is potentially confusing for users as the visual content is different to what screen readers read out.
+#### Button area might be unclear for sighted users
 
-### Next steps
-Investigate the problem of some users not seeing the plus and minus icons on the right, for example people using screen magnifiers.
+Users of speech recognition software (such as Dragon) might not know they can interact with an accordion button by referring to its heading text. Instead, the user might use the longer method of picking out its show/hide label from a list.
 
-The plus and minus icons used in this component are the most commonly used accordion controls in government services.
+While inconvenient, it’s likely the user will understand what the button area covers once they interact with it and see its focus state. [See ‘Testing with Dragon’ in the accessibility clinic summary](https://github.com/alphagov/govuk-frontend/issues/2295#issuecomment-906449543).
 
-However, more research is needed to find out how this compares to other approaches. For example [GOV.UK Step by step navigation](https://www.gov.uk/learn-to-drive-a-car) uses the words 'Show' and 'Hide' on the left side. [Check the MOT history of a vehicle](https://www.gov.uk/check-mot-history) uses up and down arrows instead of plus and minus icons.
+We want to hear about any user research teams have done with speech recognition software.
```

Comment:
@hannalaakso FYI here's the paragraphs on the Dragon issue, which is basically another version of the text in the backlog draft.